src/components/query-viewer-pivot-table/controller/query-viewer-pivot-render.tsx

```diff
@@ -107,6 +120,16 @@ export class QueryViewerPivotTableRender {
    */
   @Prop() readonly translations: QueryViewerTranslations;
 
+  /**
+   * Response Page Data
+   */
+  @Prop() readonly pageDataForTable: string;
+
+  /**
+   * Response Table Data Sync
+   */
+  @Prop() readonly getTableDataSyncXml: string;
```

Comment:
It sound like a method instead of a property. Can it be `tableDataSyncXml`?
```suggestion
  @Prop() readonly tableDataSyncXml: string;
```